cfn.yaml

```diff
@@ -59,6 +59,36 @@ Resources:
               Action:
                 - SNS:Publish
               Resource: !Ref SnsTopicForAlerts
+        - PolicyName: athenaReadAccess
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - athena:StartQueryExecution
+                - athena:GetQueryExecution
+                - athena:GetQueryResults
+              Resource: "*"
+        - PolicyName: s3CleanPageViewsAccess
+            PolicyDocument:
+              Statement:
+                Effect: Allow
+                Action:
+                  - s3:Get*
+                  - s3:List*
+                Resource: arn:aws:s3:::ophan-clean-pageview/*
+        - PolicyName: s3QueryResultsAccess
+            PolicyDocument:
+              Statement:
+                Effect: Allow
+                  Action:
+                    - s3:GetObject
+                    - s3:ListBucketMultipartUploads
+                    - s3:ListMultipartUploadParts
+                    - s3:AbortMultipartUpload
+                    - s3:PutObject
+                  Resource:
+                    - arn:aws:s3:::aws-athena-query-results-*/*'
```

Comment:
Can we store result of the query triggered by this lambda to a specific subfolder and limit access to this subfolder?

Reply:
Good idea 👍 I'll attempt to do this now...